accessibility/accessibility-in-pdfs.md

```diff
@@ -0,0 +1,94 @@
+# Accessibility in PDFs
+
+## A warning
+
+We specialise in web accessibility, not PDF accessibility. We can give you some pointers, but following this advice is **not** a guarantee that your document will be accessible. 
+
```

Comment:
```suggestion
```
Implicit. Also, given the context of the FED Playbook, who "we" are is indistinct.

Reply:
Would you still like me to remove that line now that the description has been updated to clarify the intent of the document?

Reply:
Yeah, I reckon - bearing in mind that the FED Playbook is public, I think works best with it removed.